aws/ec2.go

```diff
@@ -101,6 +99,19 @@ func getInstanceDetails(ec2Service ec2iface.EC2API, instanceID string) (*instanc
 	}, nil
 }
 
+func findFirstRunningInstance(resp *ec2.DescribeInstancesOutput) (*ec2.Instance, error) {
+	for _, reservation := range resp.Reservations {
+		for _, instance := range reservation.Instances {
+			// The low byte represents the state. The high byte is an opaque internal value
+			// and should be ignored.
+			if aws.Int64Value(instance.State.Code)&0xff == runningState {
```

Comment:
It don't think it makes a difference for this case. We're only looking at the lower byte and the MSB is completely ignored.